week5/src/sumandaverage/ScoresSumAverage.java

```diff
@@ -0,0 +1,65 @@
+package sumandaverage;
+
+import java.io.*;
+import java.net.URL;
+import java.util.Scanner;
+
+public class ScoresSumAverage {
+     public static void ReadFromWebtoFile(URL url,File F) throws IOException {
```

Comment:
this method should take url only as a paramter and return a file as the task description